docs/getting-started/install-configure.md

```diff
@@ -114,7 +114,7 @@ kubectl get all -n crossplane-system
 The Crossplane CLI extends `kubectl` with functionality to build, push, and install [Crossplane packages]:
 
 ```console
-curl -sL https://raw.githubusercontent.com/crossplane/crossplane/master/install.sh | sh
+curl -sL https://raw.githubusercontent.com/crossplane/crossplane/release-0.14/install.sh | sh
```

Comment:
does this command need a CHANNEL/VERSION too? e.g.

```
curl -sL https://raw.githubusercontent.com/crossplane/crossplane/release-0.14/install.sh | CHANNEL=alpha VERSION=v0.14.0 sh
```

Reply:
@jbw976 it will default to the appropriate :) https://github.com/crossplane/crossplane/blob/2fc6ebec4e07b518e45b7cc0aa115585d29f21d8/install.sh#L5